Neos.ContentRepository/Resources/Private/Templates/ViewHelpers/Widget/Paginate/Index.html

```diff
@@ -11,7 +11,7 @@
 <f:section name="paginator">
 	<f:if condition="{pagination.numberOfPages} > 1">
 		<nav class="page-navigation">
-			<ul class="typo3-widget-paginator">
+			<ul class="typo3-widget-paginator neos-widget-paginator">
```

Comment:
We should comment this with a "deprecated" notice. Or?
Like this: https://github.com/neos/neos-development-collection/pull/1582#issuecomment-328651117